### ISSUE TITLE
"Specs.httpd_error_log" filter changed

### ISSUE THREAD

**Issue by xiangce** [RedHatInsights/insights-frontend-assets#10]
https://github.com/RedHatInsights/insights-plugins/pull/738
~~~
ERROR_LOG = "ModSecurity: collections_remove_stale: Failed deleting collection"
add_filter(Specs.httpd_error_log, ERROR_LOG)
~~~

**Comment by kylape** [comment]
Resolved in 6e7592df

**Comment by xiangce** [comment]
Thanks.
